feature-libs/product/configuration/src/variant-configurator/interactive-configuration.module.ts

```diff
@@ -0,0 +1,119 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { CmsConfig, ConfigModule, ProductModule } from '@spartacus/core';
+import {
+  CmsPageGuard,
+  ConfigAddToCartButtonComponent,
+  ConfigFormComponent,
+  ConfigGroupMenuComponent,
+  ConfigGroupTitleComponent,
+  ConfigPreviousNextButtonsComponent,
+  ConfigPriceSummaryComponent,
+  ConfigProductTitleComponent,
+  ConfigTabBarComponent,
+  ConfigurationMessageLoaderModule,
+  DefaultMessageConfig,
+  HamburgerMenuModule,
+  PageLayoutComponent,
+} from '@spartacus/storefront';
+
+@NgModule({
+  imports: [
+    ProductModule,
+    ConfigurationMessageLoaderModule,
+    RouterModule.forChild([
+      {
+        path: 'configureCPQCONFIGURATOR/:ownerType/entityKey/:entityKey',
+        data: { pageLabel: '/configureCPQCONFIGURATOR' },
+        component: PageLayoutComponent,
+        canActivate: [CmsPageGuard],
+      },
+    ]),
+    ConfigModule.withConfig(DefaultMessageConfig),
+    ConfigModule.withConfig(<CmsConfig>{
+      cmsComponents: {
+        VariantConfigurationTabBar: {
+          component: ConfigTabBarComponent,
+          guards: [],
+        },
+        VariantConfigurationTitle: {
+          component: ConfigProductTitleComponent,
+          guards: [],
+        },
+        VariantConfigurationGroupTitle: {
+          component: ConfigGroupTitleComponent,
+          guards: [],
+        },
+        VariantConfigurationForm: {
+          component: ConfigFormComponent,
+          guards: [],
+        },
+        VariantConfigurationMenu: {
+          component: ConfigGroupMenuComponent,
+          guards: [],
+        },
+        VariantConfigurationPriceSummary: {
+          component: ConfigPriceSummaryComponent,
+          guards: [],
+        },
+        VariantConfigurationPrevNext: {
+          component: ConfigPreviousNextButtonsComponent,
+          guards: [],
+        },
+        VariantConfigurationAddToCartButton: {
+          component: ConfigAddToCartButtonComponent,
+          guards: [],
+        },
+      },
+      layoutSlots: {
```

Comment:
There's a lot in common in the `layoutSlots` of `overview.module.ts` and this one. 
Let's create a common module with the common `layoutSlots` and we can import it to both `overview.module.ts` and `interactive-configuration.module.ts`

Reply:
If it makes sense, we can also move the following CMS mappings to the common one: `ConfigTabBarComponent`, `ConfigPriceSummaryComponent`, `ConfigAddToCartButtonComponent`,

Reply:
As discussed, here I doubt that this improves readability as layouts and cms components partially deviate (and will probably do more), so the extraction of the common components e.g. would be kind of artificial and suspect to frequent changes.